README.md

```diff
@@ -18,7 +18,6 @@ be synchronised using rsync
 * api/historydata.php
 * api/keyboard.php
 * api/util.php
-* api/bundles.*
```

Comment:
What about these?
https://github.com/keymanapp/downloads.keyman.com/blob/49def4ab46215bd111747ec2de67da0bbeefd710/_tools/syncfiles.txt#L2-L4

Reply:
Good catch -- I missed these because of a save conflict on the file in vs code ...